plugins/logging/logging.cpp

```diff
@@ -21,8 +21,9 @@
 #include <ctype.h>
 #include "logging.hpp"
 #include "jlog.hpp"
+#include "roxiehelper.ipp"
 
-#define LOGGING_VERSION "LOGGING 1.0.1"
+#define LOGGING_VERSION "LOGGING 1.0.2"
 static const char * compatibleVersions[] = {
```

Comment:
@ghalliday I'm not entirely sure how the versioning works, can you comment?

Reply:
We used to use versioning on the plugins to support mixed systems and plugins that could be installed just on the eclserver and then get pushed automatically to other nodes when they needed them. We haven't supported that for a long time, and the version usage has thus rather fallen into disuse. We don't normally change the version string at all any more AFAIK

Reply:
Yes, no need to change the version number.